scripts/test-gcal-webhooks.sh

```
@@ -0,0 +1,79 @@
+  TUNNEL_URL=$(extract_url_from_log)
+  OWNED_PID=false
+else
+  echo "Starting a new Tunnelmole session..."
+  rm -f "$LOG_FILE"
+  tmole $TM_PORT > "$LOG_FILE" 2>&1 &
+  TMOLE_PID=$!
+  OWNED_PID=true
+
+  # Wait for URL or error
+  echo "Waiting for tmole to initialize..."
+  for i in {1..20}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)